_posts/2024-04-12-a-time-machine-for-your-data-opensearch-searchable-snapshots.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "A time machine for your data: OpenSearch searchable snapshots"
+authors:
+  - abunday
+date: 2024-04-10
```

Comment:
@arjunrajnetapp - Please update the date to April 15, 2024

Reply:
Thanks, the date has been updated.